ci-operator/step-registry/baremetal/lab/pre/dhcp-conf/baremetal-lab-pre-dhcp-conf-commands.sh

```diff
@@ -39,6 +42,16 @@ for bmhost in $(yq e -o=j -I=0 '.[]' "${SHARED_DIR}/hosts.yaml"); do
   fi
   DHCP_CONF="${DHCP_CONF}
 $mac,$ip,set:$CLUSTER_NAME,infinite"
+  
+  # shellcheck disable=SC2154
+  if [ -n "$IPV6_ENABLE" ]; then
```

Comment:
```suggestion
  if [ "${IPV6_ENABLE:-}" == "true" ]; then
```

Let's use true rather than any string to state IPV6 as enabled.

Also, no need for SC2154: you can define the var inline if it's not already defined as in this suggestion.